run_tests.py

```diff
@@ -7,7 +7,7 @@
 
 N_SIMPLE = 5
 N_THOROUGH = 100
-TIMEOUT = 5
+TIMEOUT = 25
```

Comment:
If your test really needs 25 seconds to complete, then something seems off. Note that the [test results from before you increased timeout](https://travis-ci.org/cahirwpz/mimiker/builds/220722118) completed successfully for many seeds, and the time required to run all tests should't differ that much from a seed to another! Examining that output I see that the main thread is in `INACTIVE` state and only the idle thread is running - so that test run would never finish successfully, no mater how much time it was given.